crowbar_framework/app/assets/stylesheets/variables.css.scss

```diff
@@ -156,17 +156,17 @@ $input-group-addon-border-color: $input-border;
 
 // Dropdowns
 // -------------------------
-$dropdown-bg: #fff;
-$dropdown-border: rgba(0,0,0,.15);
+$dropdown-bg: #25282a;
+$dropdown-border: rgba(0,0,0,.2);
```

Comment:
`.2` should be written with a leading zero as `0.2`<br>Commas in function arguments should be followed by a single space